docs/_docs/tracing.md

```diff
@@ -0,0 +1,177 @@
+---
+category: documentation
+---
+
+# Tracing
+
+## With [OpenCensus.io](https://opencenus.io) and [AWS X-ray](https://aws.amazon.com/xray/)
+
+#### Adding tracing using AWS-Xray as the exporter
+
+This example uses the AWS-Xray exporter with a global trace setting. Note that AWS X-ray exporter does not handle any metrics only tracing. 
+
+1. Add the following imports
+
+```go
+xray "contrib.go.opencensus.io/exporter/aws"
+"go.opencensus.io/plugin/ocgrpc"
+"go.opencensus.io/plugin/ochttp"
+"go.opencensus.io/trace"
+```
+
+2. Register the AWS X-ray exporter for the GRPC server
+
+```go
+xrayExporter, err := xray.NewExporter(
+    xray.WithVersion("latest"),
+    // Add your AWS region.
+    xray.WithRegion("ap-southeast-1"),
+)
+if err != nil {
+    // Handle any error.
+}
+// Do not forget to call Flush() before the application terminates.
+defer xrayExporter.Flush()
+
+// Register the trace exporter.
+trace.RegisterExporter(xrayExporter)
+```
+
+3. Add a global tracing configuration
+
+```go
+// Always trace in this example.
+// In production this can be set to a trace.ProbabilitySampler.
+trace.ApplyConfig(trace.Config{DefaultSampler: trace.AlwaysSample()})
+```
+
+4. Add `ocgrpc.ClientHandler` for tracing the grpc client calls
+
+```go
+// Example using DialContext
+conn, err := grpc.DialContext(
+    // Other options goes here.
+    // Add ocgrpc.ClientHandler for tracing the grpc client calls.
+    grpc.WithStatsHandler(&ocgrpc.ClientHandler{}),
+)
+```
+
+5. Wrap the gateway mux with the OpenCensus HTTP handler
+```go
+gwmux := runtime.NewServeMux()
+
+openCensusHandler := &ochttp.Handler{
+		Handler: gwmux,
+}
+
+gwServer := &http.Server{
+    Addr: "0.0.0.0:10000",
+    Handler: openCensusHandler,
+    }),
+}
+```
+
+#### Without a global configuration
+
+In this example we have added the [GRPC Health Checking Protocol](https://github.com/grpc/grpc/blob/master/doc/health-checking.md) and we do not wish to trace any health checks.
+
+1. Follow step `1`, `2` and `4` from the previous section
+
+2. Since we are not using a global configuration we can decide what paths we want to trace
+
+```go
+gwmux := runtime.NewServeMux()
+
+openCensusHandler := &ochttp.Handler{
+    Handler: gwmux,
+    GetStartOptions: func(r *http.Request) trace.StartOptions {
+        startOptions := trace.StartOptions{}
+        if strings.HasPrefix(r.URL.Path, "/api") {
+            // This example will always trace anything starting with /api.
+            startOptions.Sampler = trace.AlwaysSample()
+        }
+        return startOptions
+    },
+}
+```
+
+4. No global configuration means we have to use the [per span sampler](https://opencensus.io/tracing/sampling/#per-span-sampler)
+
+
+##### A method we __want__ to trace
```

Comment:
Will this be correctly rendered on the docs site? I'm not sure. Maybe best to remove the italics?

Reply:
removed it, it does not look like github does that type of bold text